website/blog/2025-07-111-train-linear-regression-model-dbt-bigframes.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "How to train a linear regression model with dbt and BigFrames"
+description: "How to build a scalable linear regression model by combining dbt's modular orchestration with BigFrames' in-database Python execution in BigQuery."
+slug: train-linear-dbt-bigframes
+authors: [jialuo_chen]
+
+tags: [analytics craft]
+hide_table_of_contents: false
+
+date: 2025-07-11
+is_featured: true
+---
+
+## Introduction to dbt and BigFrames
+
+**dbt**: A framework for transforming data in modern data warehouses using modular SQL or Python. It enables data teams to develop analytics code collaboratively and efficiently by applying software engineering best practices such as version control, modularity, portability, CI/CD, testing, and documentation. For more information, refer to [What is dbt?](/docs/introduction#dbt).
+
+> This blog post uses dbt Core (open-source, command-line interface), but you can also use dbt platform with minor configuration changes.
```

Comment:
What changes? The model code will be identical, the only thing that is different is that platform users don't need to configure their own profiles.yml file. But the prereq's say "a dbt platform or Core setup" anyway. 
```suggestion
```